spec/gitlab_custom_hooks.rb

```diff
@@ -0,0 +1,87 @@
+require 'support/publicize_methods.rb'
+require 'spec_helper'
+require 'gitlab_custom_hook'
+
+describe GitlabCustomHook do
+  let(:repository_path) { "/home/git/repositories" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.